### PR TITLE
doc typos and contributor

### PR DESCRIPTION
**Pull request by szuecs**
- fix whitespaces in code examples
- add contributor

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -241,15 +241,15 @@ Please use GitHub issues as the starting point for contributions, new ideas and/
 
 ## Contact
 
-* E-Mail: team-techmonkeys@zalando.de
-* IRC on freenode: #zalando-techmonkeys
-* Security issues: Please send an email to [maintainers](MAINTAINERS). We'll try to get back to you within two workdays. If you don't hear back, then send an email to team-techmonkeys@zalando.de.
+* E-Mail: sandor.szuecs@zalando.de
```
Comment (mikkeloscar) [inline README.md]: Would it make sense to put `team-teapot@zalando.de` like in our other projects?

Comment (szuecs) [inline README.md]: yo, done

**Comment by mikkeloscar** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**